examples/advanced/tidb-cluster.yaml

```diff
@@ -138,60 +479,161 @@ spec:
-    ## If you use require affinity, you must ensure that at least 3 nodes are available in the cluster
+    #     # require not to run tidb pods on nodes where there's already a tidb pod running
+    #     # if setting this, you must ensure that at least 3 nodes are available in the cluster
     #     requiredDuringSchedulingIgnoredDuringExecution:
     #       - labelSelector:
     #           matchExpressions:
     #             - key: app.kubernetes.io/component
     #               operator: In
     #               values:
     #                 - tidb
-    #         topologyKey: ""
-
-    ## Specify node labels for pods assignment
-    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-    # nodeSelector:
-    #   app.kubernetes.io/component: tidb
+    #         topologyKey: kubernetes.io/hostname
 
   tikv:
-    # Base image of the component
+    ############################
+    # Basic TiKV Configuration #
+    ############################
+
+    ## Base image of the component
     baseImage: pingcap/tikv
 
-    # TiKV Configuration
-    # https://pingcap.com/docs/tidb-in-kubernetes/stable/configure-cluster-using-tidbcluster/#configure-tikv-parameters
+    ## tikv-server configuration
+    ## Ref: https://docs.pingcap.com/tidb/stable/tikv-configuration-file
     config: {}
 
-    # The desired ready replicas
+    ## The desired replicas
     replicas: 3
 
-    # describes the compute resource requirements.
-    # Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    ## max inprogress failover TiKV pod counts
+    # maxFailoverCount: 3
+
+    ## describes the compute resource requirements.
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
     requests:
-      cpu: "100m"
+      cpu: 100m
+      memory: 256Mi
       storage: 1Gi
-
-    # TiDB Operator will automatically mount the certificates
-    # Defaults to "false"
-    # Ref: https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#mountclusterclientsecret
+    limits:
+      cpu: 1000m
+      memory: 2Gi
+      storage: 10Gi
+
+    ###############################
+    # Advanced TiKV Configuration #
+    ###############################
+
+    ## Overwrite TiDB cluster level configurations
+    # version: "v4.0.8"
+    # imagePullPolicy: IfNotPresent
+    # ImagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-tikv
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: tikv
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: tikv
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
```

Comment:
This sentence doesn't make much sense to me. However, if your target readers can understand it, that's okay.

Reply:
@ran-huang `environment variable` as a whole is one thing, so this sentence says that here is a list of something to set in the container